### ISSUE TITLE
Incorrect First Verse In Surah Fatihah

### ISSUE THREAD

**Issue by Skillz4Killz** [quran/quran.com-frontend#1057]
**I'm submitting a ...**

- [ ] bug report
- [ ] feature request
- [x] other

**Current behavior:**

Going to Surah Al-Fatihah Verse 1 we can see the following verse:
![image](https://user-images.githubusercontent.com/23035000/49404721-4136f380-f71e-11e8-9de4-050f40909c93.png)

Going to Surah Al-Baqarah Ayah 1, we can see the following verse:
![image](https://user-images.githubusercontent.com/23035000/49404755-5875e100-f71e-11e8-82c7-8de02d116f66.png)

The first verse of a Surah is almost never done with the Bismillah.

**Expected / new behavior:**

The Bismillah should be at the top just as it is with all other Surah's. Here is an example from Surah Al-Ma'idah

![image](https://user-images.githubusercontent.com/23035000/49404851-a854a800-f71e-11e8-8976-ced48afcc685.png)

The Bismillah is at the top but is not considered as the first verse of the Surah even though it is part of the Surah and is recited before reciting 113/114 Surahs.

**Anything else:**

Here is a hadith that helps to prove in my opinion that the first verse of Surah Fatihah is actually Alhumdulilah... and not the Bismillah.

![image](https://user-images.githubusercontent.com/23035000/49405139-958ea300-f71f-11e8-9b5c-21b38597368f.png)

https://sunnah.com/qudsi40/8

It is important to keep in mind that any Salah without the Surah Fatihah is considered as incomplete. Therefore, it is highly unlikely that Prophet Muhammad salallahu alayhi wasalam would have decided to skip the Bismillah when he was explaining the Fatihah. In my opinion, he skipped it because it is not a part of Surah Fatihah.

---

In summary, the first verse of Surah Fatihah should be changed to be Alhumdulilahi Rabilalameen and have the Bismillah displayed at the top as all other surah that have the Bismillah recited before the Surah.

**Comment by mrtugicom** [comment]
I'm in Indonesia, and that is commonly found here in many jamaah in many masjid, many schoolars already study surah Al-Fatiha, and both are allowed and tolerated, but we have one similiarity which we hold, Al-Fatiha has 7 ayah. One says Bismillah is included, and other one says not included, so some ayah seems combined or separated.
So Imam could adapt to majority by reading bismillah loudly or sirry.

To close this issue, we could give the decision to quran.com team to decide which one to use but with clear note, or leave it to user by adding option to include Bismillah in surah or not.

Wallahu'alam bishawab

**Comment by naveed-ahmad** [comment]
Salam aikum, 

All you needs to do, is just look at any Arabic mus-haf/copy of the Quran. Basmallah is listed as aya 1 of Fatihah, so we are going with the opinion of the majority of scholars, and the style of Arabic copies of the Quran.  

https://islamqa.info/en/answers/129170/the-difference-of-scholarly-opinion-as-to-whether-the-basmalah-counts-as-a-verse-of-al-faatihah-does-not-come-under-the-heading-of-distortions-of-the-quraan
